docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-254.adoc

```diff
@@ -0,0 +1,40 @@
+== Zapier webhook url detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|dc77d6d8-4dda-45e9-9fa2-5af5ae2522b7
+
+|Checkov Check ID
+|CKV_SECRET_254
+
+|Severity
+|MEDIUM
+
+|Subtype
+|build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+Exposing a Zapier webhook URL in your code can be risky. Unauthorized individuals could potentially discover this URL and use it to trigger actions in your Zapier integrations, leading to unintended consequences or data manipulation. 
+
+=== Fix - Buildtime
+
+*Zapier webhook url*
+
+To mitigate the risks associated with exposed Zapier webhook URLs:
+
+1. **Regenerate Your Webhook URL:** If you suspect your Zapier webhook URL has been compromised, regenerate it from your Zapier account. This will invalidate the old URL.
```

Comment:
**Regenerate Your Webhook URL**: Regenerate your Zapier webhook URL from your Zapier account. This will invalidate the prepvious URL.